socorro/signature/tests/test_rules.py

```diff
@@ -639,6 +639,7 @@ def test_replace_address_trailing_whitespace(self):
         },
     ],
     "threads_index": 0,
+    "thread_name": "MainThread",
```

Comment:
Same here--since we're not adjusting signature generation, we don't need to make any changes to this file.